src/lib/Conversations.ts

```diff
@@ -146,7 +146,7 @@ export default class Conversations<
     const groupsSubscription = XMTPModule.emitter.addListener(
       EventTypes.Group,
       async ({ inboxId, group }: { inboxId: string; group: GroupParams }) => {
-        if (this.known[group.id] || this.client.inboxId !== inboxId) {
```

Comment:
The forked group test is still flaky for me so I don't think this fixes it. I'd probably lean more towards reverting this change just in case. But also okay with leaving it if you think it will fix other issues as all the tests seem to be passing.